libs/common-services/src/main/java/uk/gov/di/ipv/core/library/tracing/TracingHttpClient.java

```diff
@@ -46,7 +48,10 @@ private TracingHttpClient(HttpClient baseClient) {
     }
 
     public static HttpClient newHttpClient() {
-        return new TracingHttpClient(HttpClient.newHttpClient());
+        return new TracingHttpClient(
+                JavaHttpClientTelemetry.builder(GlobalOpenTelemetry.get())
+                        .build()
+                        .newHttpClient(HttpClient.newHttpClient()));
```

Comment:
Does this cause any issues in environments without Dynatrace - i.e. local/dev. The API tests failed, but not sure if that's really related :/

Reply:
Doesn't seem to - API tests are green, and it seems happy in my dev env deployed without dynatrace